xdg-ninja.sh

```diff
@@ -168,19 +212,24 @@ check_file() {
     FILENAME="$2"
     MOVABLE="$3"
     HELP="$4"
+    JSON_FILE="$5"
```

Comment:
I think, there should be separate `check_file` function for json files, since output format is totally different

Reply:
While the output format is totally different most of the logic is still the exact same.